Rtp/RtpClient.Methods.cs

```diff
@@ -2201,7 +2200,7 @@ void HandleEvents()
                         HandleEvent();
                     }
                 }
-                catch (System.Threading.ThreadAbortException) { System.Threading.Thread.ResetAbort(); Media.Common.ILoggingExtensions.Log(Logger, ToString() + "@HandleEvents Aborted"); }
+                catch (System.Threading.ThreadAbortException) { Media.Common.ILoggingExtensions.Log(Logger, ToString() + "@HandleEvents Aborted"); }
```

Comment:
I don't understand why we remove the call to ResetAbort? but we still catch the same exception.

Reply:
If we are going to keep the catching of the AbortException I prefer to keep ResetAbort code.

Reply:
`ResetAbort` throws `PlatformNotSupportedException` in .net core.

Reply:
But you also can't catch ThreadAbortException (from the system), I think we should move or better comment the entire catch block for ThreadAbortException out if we are removing it.